### ISSUE TITLE
Tuning parameters for physical experiment?

### ISSUE THREAD

**Issue by aarontan-git** [idincern/idincern-husky#1]
Hello,

Have you tried physical experiments?

It seems that the husky behave very differently in reality when compared to gazebo simulation. I read online that there are parameters that should be tuned; however, I am not sure which ones.

Can you help?

Thanks,
Aaron

**Comment by idincern** [comment]
No, we haven't tried all of the planners in real husky. But the default husky planners&parameters were working fine on the real husky. I assume they were A* and DWA. You may also tune the parameters inside costmap_common.yaml & costmap_local.yaml files as you wish.
